functions/_nvm_auto_invoke.fish

```diff
@@ -1,5 +1,8 @@
 function _nvm_auto_invoke \
```

Comment:
Perhaps a better name is `_nvm_use_on_pwd_change` and `$nvm_use_on_pwd_change`. I'm open to discuss that.